### PR TITLE
chore: add deprecation notices

### PR DESCRIPTION
**Pull request by noahwaldner**

### CODE REVIEW

File: packages/fondue/src/components/Tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -56,6 +56,9 @@ const handleTimeout = (callback: () => void, delay: number, timeoutId: number |
 
 const isPointerEventTypeMouse = (event: PointerEvent): boolean => event.pointerType === 'mouse';
 
+/**
+ * @deprecated Please use updated tooltip component from `@frontify/fondue/components`.
```
Comment (SamuelAlev) [inline packages/fondue/src/components/Tooltip/Tooltip.tsx:60]: Could you add the migration guide like the other `@deprecated`?

File: packages/fondue/src/components/Tooltip/InContextTooltip.stories.tsx [hunk 2]
```diff
@@ -16,9 +16,17 @@ import IconIcon24 from '@foundation/Icon/Generated/IconIcon24';
 import IconJohanna from '@foundation/Icon/Generated/IconJohanna';
 import IconJohanna24 from '@foundation/Icon/Generated/IconJohanna24';
 
+/**
+ ### This component is deprecated, please use the [new Tooltip](/docs/current_components-tooltip--documentation) component instead.
```
Comment (SamuelAlev) [inline packages/fondue/src/components/Tooltip/InContextTooltip.stories.tsx:20]: Same for the rest
```suggestion
 ### This component is deprecated, please use the [new Tooltip component](/docs/current_components-tooltip--documentation) instead.
```

File: packages/fondue/src/components/LegacyTooltip/LegacyTooltip.tsx [hunk 3]
```diff
@@ -143,6 +143,9 @@ const TooltipWrapper = ({
     return children as JSX.Element;
 };
 
+/**
+ * @deprecated Please use updated tooltip component from `@frontify/fondue/components`.
```
Comment (SamuelAlev) [inline packages/fondue/src/components/LegacyTooltip/LegacyTooltip.tsx:147]: Same (and all other files)

File: packages/fondue/src/components/Tooltip/Tooltip.tsx (hunk 1 above)
Comment (noahwaldner) [reply packages/fondue/src/components/Tooltip/Tooltip.tsx]: added

File: packages/fondue/src/components/LegacyTooltip/LegacyTooltip.tsx (hunk 3 above)
Comment (noahwaldner) [reply packages/fondue/src/components/LegacyTooltip/LegacyTooltip.tsx]: added

**Review by SamuelAlev — APPROVED** [review]

**Pull request merged**